### PR TITLE
[javascript] integration with dap layer

### PR DESCRIPTION
**Pull request by yyoncho**
- Fixed also instructions on language server selection

### CODE REVIEW

File: CHANGELOG.develop [hunk 1]
```diff
@@ -977,6 +977,7 @@ Other:
   - Control auto-newline using the =c-c++-enable-auto-newline= variable
     (thanks to Fangrui Song and Codruț Constantin Gușoi)
   - Added =org-babel= support (thanks to Michael Rohleder)
+  - Added debug support via =dap= layer (Firefox and Google Chrome)
```
Comment (smile13241324) [inline CHANGELOG.develop]: Just noticed that this line is in the wrong group, should be in the javascript section.

File: layers/+lang/javascript/README.org [hunk 2]
```diff
@@ -111,11 +113,16 @@ Formatter can be chosen on a per project basis using directory local variables
...
-
+You have to install =typescript-language-server= (recommended) or =javascript-typescript-langserver= language server packages via
+#+BEGIN_SRC sh
+  npm i -g typescript typescript-language-server
+#+END_SRC
+or
 #+BEGIN_SRC sh
   npm i -g typescript javascript-typescript-langserver
 #+END_SRC
+*** Debugger (dap integration)
```
Comment (smile13241324) [inline layers/+lang/javascript/README.org]: Please add a blank line between the headline and the text before.

File: layers/+lang/javascript/README.org [hunk 3]
```diff
@@ -254,3 +261,48 @@ You can check more [[https://github.com/mooz/js-doc/][here]]
 | ~SPC m s r~ | send current region to the skewer REPL                                             |
 | ~SPC m s R~ | send current region to the skewer REPL and switch to it in insert state            |
 | ~SPC m s s~ | switch to REPL                                                                     |
+** debugger (dap mode)
```
Comment (smile13241324) [inline layers/+lang/javascript/README.org]: Please add a blank line between the headline and the text before.

**Comment by yyoncho** [comment]
Done.

**Comment by smile13241324** [comment]
Thank you for contributing to Spacemacs! :+1:
The PR has been cherry-picked into develop, you can safely delete your branch.

**Pull request closed without merging**